### DISCUSSION TITLE
Closing application keeps moonlight <--> sunshine connection open

### DISCUSSION THREAD

**Discussion by Zobby31** [LizardByte/community#132] [Sunshine]
Hello guys,

First time posting here so please excuse me if I missed something that already exists or if it is not the appropriate place.

I was wondering whether it is possible or not to :

• kill the connection between sunshine and moonlight when closing an application
• close the launched application when terminating the connection between sunshine and moonlight manually

As for me, when I exit a game by selecting "Exit to desktop", the connection between sunshine and moonlight remains open and I keep seeing the desktop in moonlight.

On the other hand, when I manually stop the moonlight connection, the game remains up and running on the host.

As far as I remember, it was different with GeForce Experience.

Am I doing something wrong?

Thanks for your help.

**Comment by pedromvu** [comment]
I also found this is the case, I assume expected?

The way I do it with Steam, is that I added command preparations in Applications config

![image](https://user-images.githubusercontent.com/22568427/214739901-e98f4844-6401-4e06-ba13-ef8473f0c330.png)

But need to make sure to not select Exit directly from Steam, instead I go to my Host Home, back into Moonlight and select the app then running app then Exit.

Before adding the command the host would simply keep running Steam, but now is closed.

[a comment by an automated account is not shown]
